### PR TITLE
Add testing with Jest

### PR DESCRIPTION
**Pull request by jnv**

### CODE REVIEW

**Review by marianhlavac — APPROVED** [review]
awesome 😍

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (marianhlavac) [inline .babelrc:1]: WHY? This one doesn't work with Ream.

Comment (jnv) [reply .babelrc:1]: Not with the old version of Ream you have included, anyway. ;)

Comment (marianhlavac) [reply .babelrc:1]: i have included the latest version of ream atm, not my fault it updates so often.

Comment (jnv) [reply .babelrc:1]: Yeah, beta is being suggested only for 3 days now: https://github.com/ream/ream#install

**Pull request merged**